### PR TITLE
Small change to login + testing helper class.

### PR DESCRIPTION
**Pull request by sduoase**
Remove login url encoded message.
Successful register automatically logins the user.
Create a testing helper class that provides functin for login, logout, create user. It must be subclassed by real testing classes.

### CODE REVIEW

**Comment by sduoase** [comment]
I don't know if we should just provide the simple variants like _login(...) and _create_user(...) and then let the user do the success and failure.

The pros of having the success and failure variants is that we probably want to have successful logins etc in more than 1 part of our code.

The cons is that checking success or failure there might create a structure which is too rigid.

**Comment by sduoase** [comment]
Anyway, my idea is that as we expand the testing we will add here functions to create stories etc.

Basically every action that is required more than 1 time should be implemented here.

File: monolith/views/auth.py [hunk 1]
```diff
@@ -39,14 +36,14 @@ def create_user():
...
-    if request.method == 'POST' and form.validate_on_submit():
+    if form.validate_on_submit():
         new_user = User()
         form.populate_obj(new_user)
         new_user.set_password(form.password.data)
         db.session.add(new_user)
         try:
             db.session.commit()
-            return redirect(url_for("auth.login", message="You have been sucessfully registered!"))
+            return login()
```
Comment (alessandrobelfiore) [inline monolith/views/auth.py]: Maybe add message to help testing?

File: monolith/views/auth.py [hunk 2]
```diff
@@ -39,14 +36,14 @@ def create_user():
     if not current_user.is_anonymous:
         return redirect("/", code=302)
```
Comment (alessandrobelfiore) [inline monolith/views/auth.py]: Maybe add message to help testing?

Comment (sduoase) [reply monolith/views/auth.py]: IMHO, this is more of a quality of life feature, not something we really have to test.

I would stick to testing the major features offered by the website, and ignore the small details

**Pull request merged**